src/main/webapp/assets/college-locations.json

```diff
@@ -0,0 +1,77 @@
+[
+  {
+    "UNITID": "209542",
+    "NAME": "Oregon State University",
+    "STREET": "1500 SW Jefferson Avenue",
+    "CITY": "Corvallis",
+    "STATE": "OR",
+    "ZIP": "97331",
+    "STFIP": "41",
+    "CNTY": "41003",
+    "NMCNTY": "Benton County",
+    "LOCALE": "13",
+    "LAT": "44.56395",
+    "LON": "-123.274723",
+    "CBSA": "18700",
+    "NMCBSA": "Corvallis, OR",
+    "CBSATYPE": "1",
+    "CSA": "440",
+    "NMCSA": "Portland-Vancouver-Salem, OR-WA",
+    "NECTA": "N",
+    "NMNECTA": "N",
+    "CD": "4104",
+    "SLDL": "41016",
+    "SLDU": "41008",
+    "SCHOOLYEAR": "2019-2020"
+  },
+  {
+    "UNITID": "122931",
+    "NAME": "Santa Clara University",
+    "STREET": "500 El Camino Real",
+    "CITY": "Santa Clara",
+    "STATE": "CA",
+    "ZIP": "95053",
+    "STFIP": "06",
+    "CNTY": "06085",
+    "NMCNTY": "Santa Clara County",
+    "LOCALE": "12",
+    "LAT": "37.348362",
+    "LON": "-121.93784",
+    "CBSA": "41940",
+    "NMCBSA": "San Jose-Sunnyvale-Santa Clara, CA",
+    "CBSATYPE": "1",
+    "CSA": "488",
+    "NMCSA": "San Jose-San Francisco-Oakland, CA",
+    "NECTA": "N",
+    "NMNECTA": "N",
+    "CD": "0617",
+    "SLDL": "06025",
+    "SLDU": "06010",
+    "SCHOOLYEAR": "2019-2020"
+  },
+  {
+    "UNITID": "110653",
+    "NAME": "University of California-Irvine",
+    "STREET": "501 Aldrich Hall",
+    "CITY": "Irvine",
+    "STATE": "CA",
+    "ZIP": "92697",
+    "STFIP": "06",
+    "CNTY": "06059",
+    "NMCNTY": "Orange County",
+    "LOCALE": "11",
+    "LAT": "33.648434",
+    "LON": "-117.841248",
+    "CBSA": "31080",
+    "NMCBSA": "Los Angeles-Long Beach-Anaheim, CA",
+    "CBSATYPE": "1",
+    "CSA": "348",
+    "NMCSA": "Los Angeles-Long Beach, CA",
+    "NECTA": "N",
+    "NMNECTA": "N",
+    "CD": "0645",
+    "SLDL": "06074",
+    "SLDU": "06037",
+    "SCHOOLYEAR": "2019-2020"
+  }
+]
```

Comment:
Oh cool! So we can now just use the "UNITID" as the entity name in Datastore -- love it! Also new line

Reply:
Yeah! They're guaranteed to be unique within the U.S. and we don't even have to store the colleges themselves in the Datastore, just associate a post with a given UNITID and viola! (also new line fixed!)